mne/viz/tests/test_topomap.py

```diff
@@ -224,7 +244,3 @@ def test_plot_tfr_topomap():
     _onselect(eclick, erelease, tfr, pos, 'mag', 1, 3, 1, 3, 'RdBu_r', list())
     tfr._onselect(eclick, erelease, None, 'mean', None)
     plt.close('all')
-
-
-def test_prepare_topo_plot():
-    """Test obtaining 2D coordinates from 3D sensor locations"""
```

Comment:
because the file ends there. There is no test. I wanted to write one, but never got around to it. It was mistakenly committed.